knexfile.js

```diff
@@ -12,13 +12,22 @@ module.exports = {
       port : 3306,
       user : process.env.MYSQL_USERNAME,
       password : process.env.MYSQL_PASSWORD,
-      database : process.env.MYSQL_DATABASE
+      database : process.env.MYSQL_DATABASE,
+      socketPath: '/var/run/mysqld/mysqld.sock'
     }
   },
   staging: {
     
   },
   production: {
-    
+    client: 'mysql',
+    connection: {
+      host : '127.0.0.1',
+      port : 3306,
+      user : process.env.MYSQL_USERNAME,
+      password : process.env.MYSQL_PASSWORD,
+      database : process.env.MYSQL_DATABASE,
+      socketPath: '/var/run/mysqld/mysqld.sock'
```

Comment:
socket won't be used for production

Reply:
Not sure why but without it, the app won't work, it can't connect to MySQL database. I don't have much experience with MySQL for NodeJS app. Can you give some advice ?

Reply:
Let check it. Not sure if it works or not